Assets/Scripts/UI/DraggableObject.cs

```diff
@@ -121,6 +128,14 @@ public virtual void OnEndDrag(PointerEventData eventData)
         }
     }
 
+    public void tappingEffect(PointerEventData eventData)
```

Comment:
This method duplicates code from ScreenTapping.cs.

Might be better to try and find a way to call ScreenTapping's method directly rather than have our own duplicate method.